snap/snapcraft.yaml

```diff
@@ -90,6 +90,11 @@ parts:
       gunzip "./$PACKAGE.gz"
       tar xvf "./$PACKAGE" --strip-components=1
       popd
+    stage:
```

Comment:
Note: these changes are from the same commits as in https://github.com/edgexfoundry/edgex-go/pull/331
Not a blocker, just pointing this out